examples/hello_world_serde.rs

```diff
@@ -0,0 +1,75 @@
+use bevy::prelude::*;
+use bevy_input_actionmap::*;
+
+fn main() {
+    App::build()
+    .add_plugins(DefaultPlugins)
+    .add_plugin(ActionPlugin::<Action>::default())
+    .add_startup_system(setup.system())
+    .add_system(run_commands.system())
+    .run();
+}
+
+#[derive(Hash, PartialEq, Eq, Clone)]
+#[cfg_attr(feature = "serialize", derive(Serialize, Deserialize))]
+enum Action {
+    Select,
+    SuperSelect,
+    AwesomeSuperSelect,
+}
+
+fn setup(mut input: ResMut<InputMap<Action>>){
+    #[cfg(feature = "serialize")]
+    if let Err(_) = load_from_path(&mut input, "keybindings.config") {
```

Comment:
The file extension should be .ron for Rusty Object Notation files. Also a configurable file path would be great if you want to support people having assets in different places, configs in OS-specific locations like ~/.config/ or C:\Users\Example\AppData\Roaming\ etc.